src/q1/analyse-EPL1102/summary/analyse-EPL1102-summary.tex

```diff
@@ -0,0 +1,993 @@
+% !TEX encoding = UTF-8 Unicode
+\documentclass[fr]{../../../eplsummary}
+\usepackage{setspace}
+\usepackage{framed}
+\usepackage{pgfplots}
+
+\hypertitle{analyse}{1}{EPL}{1102}{Philippe Dan}{Jean-François Remacle et François Glineur}
```

Comment:
```suggestion
\hypertitle[']{Analyse}{1}{EPL}{1102}{Philippe Dan}{Jean-François Remacle et François Glineur}
```

Reply:
Remacle en première? J'ai pitié d'eux...

Reply:
Il manque celle ci :)

Reply:
This one is alone 😢 , @pdan1

Reply:
```suggestion
\hypertitle[']{Analyse}{1}{EPL}{1102}{Philippe Dan}{Jean-François Remacle et François Glineur}
```